src/main/java/ladder/LadderMain.java

```diff
@@ -0,0 +1,15 @@
+package ladder;
+
+import ladder.controller.LadderController;
+import ladder.view.InputView;
+import ladder.view.ResultView;
+
+public class LadderMain {
+    private static final InputView INPUT_VIEW = new InputView();
+    private static final ResultView RESULT_VIEW = new ResultView();
+
+    public static void main(String args[]){
```

Comment:
```suggestion
    public static void main(String[] args){
```
메인메서브드는 대표적으로 템플릿은 이런 형태입니다.